sources/packages/backend/libs/integrations/src/services/student/student.service.ts

```diff
@@ -145,6 +106,50 @@ export class StudentService extends RecordDataModelService<Student> {
     return studentRepo.save(studentToUpdate);
   }
 
+  /**
+   * Get student by SIN, Last name and birth date.
+   * @param sin sin.
+   * @param lastName last name.
+   * @param birthDate birth date.
+   * @returns student.
+   */
+  async getStudentBySINAndLastNameAndBirthDate(
+    sin: string,
+    lastName: string,
+    birthDate: string,
+  ): Promise<Student> {
+    return this.repo.findOne({
+      select: { id: true, disabilityStatus: true },
+      where: {
+        sinValidation: { sin },
+        user: {
+          lastName: Raw((alias) => `LOWER(${alias}) = LOWER(:lastName)`, {
```

Comment:
just wondering, we can use https://orkhan.gitbook.io/typeorm/docs/find-options#:~:text=LIKE%20%27%25out%20%23%25%27-,ILike,-import%20%7B%20ILike

Reply:
IMO, by using the lower function, we use the `=` operator which is more efficient than like or wild card operator.

Reply:
I see the point that using ilike without wildcard and `=` could be potentially have same efficiency. 

But I would like it keep as it is.